src/Project.php

```diff
@@ -2462,42 +2462,52 @@ public static function showKanban($ID)
             ],
             'supported_filters'           => [
                 'title' => [
+                    'label' => __('title'),
```

Comment:
In this file too, context should be added.

```suggestion
                    'label' => _x('filters', 'title'),
```